package.json

```diff
@@ -10,6 +10,7 @@
     "lint:js": "eslint --ext .js,.svelte .",
     "lint:shell": "shellcheck all create",
     "storybook": "start-storybook -s ./public -p 6006",
+    "deploy-storybook": "storybook-to-ghpages",
```

Comment:
@openjck On 2nd thought, perhaps this makes more sense as part of the github actions installation step? Unless you think some devs would want to deploy manually, I can move it there.

Reply:
Oops, meant to comment on the dependency, not the run command. But this would go away too.